syntax="proto3";

package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns(HelloReply) {}
}

message HelloRequest {
  string message = 1;
}

message HelloReply {
  string message = 1;
}

// protoc --cpp_out=./ helloworld.proto 
// protoc --grpc_out=. --plugin=protoc-gen-grpc=/mnt/d/Code/see/src/thirdparty/download/grpc/grpc_cpp_plugin helloworld.proto